### DISCUSSION TITLE
Something went wrong

### DISCUSSION THREAD

**Discussion by Rez-21** [NREL/REopt-API-Analysis#167] [1 - General REopt Questions]
Can anyone help me with this error? Do you know if this is a general problem with REopt?
![qqqq](https://github.com/user-attachments/assets/c1e230b3-fd18-4179-b772-7677525a75ac)

**Comment by adfarth** [comment]
Hi, could you share the inputs that you changed before you got this error message? This typically appears when the inputs create an infeasible optimization problem.

**Reply by Rez-21** [reply]
Hey. The design includes PV (min 40 MW and max 200 MW, costs and installation parameters been modified), BESS (costs, min and initial SoC modified), generator with a nominal capacity of 13.165 MW and set the min and max power the same value to force the optimization problem to the fixed sized generation and min loading of 100%. Also, load profile was uploaded as below:
![image](https://github.com/user-attachments/assets/22c39f40-e3c0-4692-b04b-16dc9f72e918)

**Reply by Rez-21** [reply]
I just removed the prime generator constraints (min and max limits) and got the following error:
![image](https://github.com/user-attachments/assets/3ee394be-e65f-486a-82d4-31875b45625b)
